managed/src/main/java/com/yugabyte/yw/common/KubernetesManager.java

```diff
@@ -105,6 +107,24 @@ public ShellResponse getServiceIPs(Map<String, String> config,
     return execCommand(config, commandList);
   }
 
+  public ShellResponse getNodeInfos(Map<String, String> config) {
```

Comment:
would be better to make this command return json directly instead of a ShellResponse object, which isn't very meaningful to the caller.

Reply:
The main reason to keep it that way was to make it consistent with rest of the code in this file. That way we can re-use the same function somewhere else if we have to. For example, using the getSecret at some other place which might have different way to handle the errors, than the current caller (which we can still do by catching exceptions). What do you think?

Reply:
Agree that the rest of the code returns ShellResponse but that is not ideal too. It should be ok to return a JsonNode directly and throw exceptions to communicate this to the caller. One option is to have two methods getNodeInfos -> JsonNode which calls the runGetNodeInfos -> ShellResponse

Reply:
I have made the changes. Let me know how do you find them.